src/LibVLCSharp.WPF/ForegroundWindow.cs

```diff
@@ -109,35 +102,90 @@ void Background_Loaded(object? sender, RoutedEventArgs e)
             }
         }
 
-        void RefreshOverlayPosition(object? sender, EventArgs e)
+        void Bckgnd_LayoutUpdated(object? sender, EventArgs e)
         {
-            if (PresentationSource.FromVisual(_bckgnd) == null)
```

Comment:
@mfkl: I unintentionally removed this protection during my DRY refactoring. I am creating a PR to restore it.